sickbeard/tvcache.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, providerName):
             if not self.hasTable(providerName):
                 logger.log(u"Creating cache table for provider {}".format(providerName), logger.DEBUG)
                 self.action(
-                    "CREATE TABLE [" + providerName + "] (name TEXT, season NUMERIC, episodes TEXT, indexerid NUMERIC, url TEXT, time NUMERIC, quality TEXT, release_group TEXT)")
+                    "CREATE TABLE [" + providerName + "] (name TEXT, season NUMERIC, episodes TEXT, indexerid NUMERIC, url TEXT, time NUMERIC, quality NUMERIC, release_group TEXT)")
```

Comment:
@p0psicles testing in pubdate2 but I wont delete my cache.db so this wont change for me